src/components/ListHeader.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <header class="uk-background-primary uk-padding-small uk-flex uk-flex-middle uk-flex-between">
+    <!-- TODO: Use icon instead of "Home" text -->
+    <oc-breadcrumb class="oc-light" :items="breadcrumbsItems" />
+    <div>
+      <oc-button
+        variation="primary"
+        :disabled="!isSelectBtnEnabled"
+        :uk-tooltip="disabledSelectBtnTooltip"
+        @click.native="select"
+      >
+        {{ isLocationPicker ? 'Select location' : 'Select resources' }}
```

Comment:
I guess translation is something for later. Raise ticket?

Another approach would be to make these texts configurable from the outside

Reply:
Ticket created #7